docs/README.rst

```diff
@@ -726,6 +728,7 @@ OpenShift
 .. rubric:: **Footnotes**
 .. [#objectpartition] The |kctlr| creates and manages objects in the BIG-IP partition defined in the `F5 resource`_ ConfigMap. **It cannot manage objects in the** ``/Common`` **partition**.
 .. [#nodeportmode] The |kctlr| forwards traffic to the NodePort assigned to the Service by Kubernetes. See the `Kubernetes Service`_ documentation for more information.
+.. [#whitelist] If the user desires to customize a virtual server resource, the user must indicated this by adding a whitelist metadata flag to the resource.  The metadata flag changes the behavior of the controller as it interacts with Big-IP resources.  The controller will not delete whitelisted resources.  When the controllers needs to modify a resource, instead of replacing the resource settings with the desired controller settings, the controller will merge the settings into the existing resource. The user must be aware of potential conflicts that can arise under this mode of operation. You can find further details of this feature in the |kctlr-long| `user documentation`_.
```

Comment:
- "the user" --> "you"
- "controller" --> "Controller"

- "modify the controller's behavior so that it merges" --> "set the Controller to merge"

- I'd actually take out the whole footnote and just add a link to the solution docs at line 69: 
```
"See `Manage OpenShift Routes`_ for more information."
```

- Add the following to the rst_epilog section of docs/conf.py:
```
.. _Manage OpenShift Routes:  %(base_url)s/containers/latest/openshift/kctlr-openshift-routes.html
```

Reply:
Awesome. Please make the requested wording changes and then we'll be good to go.

Reply:
Please refresh your browser.  The footnote is gone (unless you are talking about some other wording).